nomad/assets/dashboards/overview.json

```diff
@@ -0,0 +1,978 @@
+                        "q": "(avg:nomad.client.host.cpu.user{$host,$node_id,$datacenter}+avg:nomad.client.host.cpu.system{$host,$node_id,$datacenter})"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "CPU Usage",
+            "type": "query_value",
+            "width": 20,
+            "x": 72,
+            "y": 0
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "custom_unit": "%",
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "(avg:nomad.client.host.disk.used{$host,$node_id,$datacenter}/avg:nomad.client.host.disk.size{$host,$node_id,$datacenter})*100"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "Disk Usage",
+            "type": "query_value",
+            "width": 20,
+            "x": 93,
+            "y": 0
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "14",
+            "height": 5,
+            "html": "Broker Metrics",
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 18,
+            "x": 75,
+            "y": 52
+        },
+        {
+            "height": 8,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "sum",
+                        "q": "avg:nomad.nomad.broker.total_ready{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "Total Brokers Ready",
+            "type": "query_value",
+            "width": 18,
+            "x": 75,
+            "y": 57
+        },
+        {
+            "height": 8,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "sum",
+                        "q": "avg:nomad.nomad.broker.total_blocked{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "Total Brokers Blocked",
```

Comment:
Can we have 1 pixel between each widget and label for `Broker Metrics` and `RPC Metrics`? 

https://datadoghq.dev/integrations-core/guidelines/dashboards/#widgets